types/testing-library__cypress/tslint.json

```diff
@@ -1,10 +1,6 @@
 {
     "extends": "dtslint/dt.json",
     "rules": {
-        "semicolon": [
-            true,
-            "never"
-        ],
```

Comment:
Apologies for removing the lint rule that disables semicolons, but since [the 14th of June](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/35672 ) `prettier` has been automatically formatting changes to include semicolons (so subsequently it would fail the lint rules that had been setup) and I didn't think it was a good idea to have a lint rule which fought the global configuration of `prettier`.

Reply:
sounds good! just one minor non-blocking comment - would be great to have the prettier config change and prettifying the code as 2 separate commits (and maybe in separate PR). And the `configure` export change on top of that. This would make the reviewing faster and easier 👍

Reply:
@airato It's two separate commits within this PR now.

Reply:
thank you!